tensorflow/core/framework/function_testlib.h

```diff
@@ -40,6 +40,14 @@ class Attrs {
     }
   }
 
+  Attrs(
```

Comment:
why is this needed?

Reply:
Some places like [here](https://github.com/tensorflow/tensorflow/pull/31635/files#diff-ba4d21ca0c437c902fd8c791e06ba44dR308) need it to convert [AttributeVector](https://github.com/tensorflow/tensorflow/pull/31635/files#diff-a4a3b8bfea4ac6f0185c2a132dadbb8bR32) to `Attrs`.

Reply:
`AttributeVector` can not be replaced by `Attrs` as it is required by `test::function::NDef` like [here](https://github.com/tensorflow/tensorflow/pull/31635/files#diff-4085eb633f02b68967febffed95bc6edR41).

Reply:
ack